.jscsrc

```diff
@@ -1,8 +1,6 @@
 {
   "preset": "google",
   "disallowSpacesInAnonymousFunctionExpression": null,
-  "excludeFiles": [
-    "node_modules/**",
-    "gulpfile.babel.js"
-  ]
+  "esnext": true,
+  "excludeFiles": ["node_modules/**"]
```

Comment:
jscs excludes `node_modules` by default now (although it looks like we need to update the docs on this: http://jscs.info/overview.html#excludefiles, which I'l do now)